src/picongpu/include/traits/PICToAdios.tpp

```diff
@@ -32,7 +34,7 @@ namespace traits
 {
     /** Trait for int */
     template<>
-    struct PICToAdios<int>
```

Comment:
I mean not a comment for the user but a comment for developer that we not add types like int or unsigned long in a half year because we forget that it can conflict with uint32_t.
This double definition can compile but if a user switch to a a tegra or similar maybe it crash.